src/tilemaps/mapdata/MapData.js

```diff
@@ -145,7 +172,7 @@ var MapData = new Class({
          * An array with all the layers configured to the MapData.
          * 
          * @name Phaser.Tilemaps.MapData#layers
-         * @type {array}
+         * @type {Array.<LayerData|ObjectLayer>}
```

Comment:
Typed arrays in jsdoc are represented like this: `LayerData[]` and when you've got a type that can be more than one thing you wrap them in brackets, so: `{(LayerData[]|ObjectLayer)}`.